tests/test_e2e_13_mef_eline.py

```diff
@@ -0,0 +1,1117 @@
+        response = requests.patch(api_url + evc1, data=json.dumps(payload2),
+                                  headers={'Content-type': 'application/json'})
+        assert response.status_code == 400
+        time.sleep(10)
+
+        # It verifies EVC's data
+        response = requests.get(api_url + evc1)
+        data = response.json()
+        assert data['backup_links'][0]['endpoint_a']['id'] == payload1['backup_links'][0]['endpoint_a']['id']
+        assert data['backup_links'][0]['endpoint_b']['id'] == payload1['backup_links'][0]['endpoint_b']['id']
+        assert data['active'] is True
+
+    """It is returning Response [500], should be 400"""
+    @pytest.mark.xfail
+    def test_145_patch_an_unrelated_backup_links(self):
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        payload1 = {
+            "name": "my evc1",
+            "enabled": True,
+            "uni_a": {
+                "interface_id": "00:00:00:00:00:00:00:01:1",
+            },
+            "uni_z": {
+                "interface_id": "00:00:00:00:00:00:00:02:1"
+            },
+            "backup_links": [
+                {"endpoint_a": {"id": "00:00:00:00:00:00:00:02:3"},
+                 "endpoint_b": {"id": "00:00:00:00:00:00:00:03:2"}}
+            ]
+        }
+        response = requests.post(api_url, data=json.dumps(payload1),
+                                 headers={'Content-type': 'application/json'})
+        data = response.json()
+        evc1 = data['circuit_id']
+
+        payload2 = {
+            "backup_links": [
+                {"endpoint_a": {"id": "00:00:00:00:00:00:00:01:1"},
+                 "endpoint_b": {"id": "00:00:00:00:00:00:00:02:1"}}
+            ]
+        }
+
+        # It sets a new circuit's primary_path
+        response = requests.patch(api_url + evc1, data=json.dumps(payload2),
+                                  headers={'Content-type': 'application/json'})
+        assert response.status_code == 400
+        time.sleep(10)
+
+        # It verifies EVC's data
+        response = requests.get(api_url + evc1)
+        data = response.json()
+        assert data['backup_links'][0]['endpoint_a']['id'] == payload1['backup_links'][0]['endpoint_a']['id']
+        assert data['backup_links'][0]['endpoint_b']['id'] == payload1['backup_links'][0]['endpoint_b']['id']
+        assert data['active'] is True
+
+    """It is returning Response 500, but it should return a failure code"""
+    @pytest.mark.xfail
+    def test_150_patch_creation_time(self):
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        evc1 = self.create_evc(100)
+
+        start = datetime.now()
+        payload = {
+            "creation_time": start.strftime(TIME_FMT)
+        }
+
+        # It sets a new circuit's creation_time
+        response = requests.patch(api_url + evc1, data=json.dumps(payload),
+                                  headers={'Content-type': 'application/json'})
+        assert response.status_code == 400
```

Comment:
add an assertion to make sure the creation_time was not changed from the original request

Reply:
It is resolved with commit [dd2c2ec](https://github.com/amlight/kytos-end-to-end-tests/pull/69/commits/dd2c2ecfd5e157d55020f2a90431992fdbc91369)